canonical-web-and-design/snapcraft.io
# Pin pull licenses from spdx repo

## Done
- Pin spdx licenses
- Remove duplicate custom licenses that didn't exist on the previous version of the license file
- This will help aligning the licenses across store/snapd/snapcraft

## How to QA
- `dotrun clean`
- Visit some snap details pages and make sure the licenses are correct
- Visit some listing pages and make sure the licenses match the details pages and that nothign has changed

## Issue / Card
Fixes #3451
